lib/tlog/rec_session_conf.c

```diff
@@ -342,9 +348,15 @@ tlog_rec_session_conf_get_shell(struct tlog_errs **perrs,
         grc = TLOG_RC_FAILURE;
         TLOG_ERRS_RAISES("Command string is not specified");
     }
+    if (command && interactive) {
+        grc = TLOG_RC_FAILURE;
+        TLOG_ERRS_RAISES("Specified arguments cannot be used together");
+    }
```

Comment:
I tried various shells, and it seems that both '-c' and '-i' can be used together, so tlog shouldn't put these restrictions on its interface.

Reply:
Thanks Nikolai, I will fix it